blob/blob.go

```diff
@@ -66,6 +66,10 @@
 //   are included because they call NewRangeReader.)
 // - NewWriter, from creation until the call to Close.
 //
+// It also collects the following metrics:
```

Comment:
Not part of this PR, but should the section above say what the name of the metric is for each method?

Reply:
I just did that for pubsub in #1232. I kept the wording generic so we can cut and paste. If you like that, I'll repeat it here and elsewhere.